client/components/common/tce-core/Discussion/index.vue

```diff
@@ -97,7 +97,7 @@ export default {
     hasHiddenComments: vm => vm.commentsShownLimit < vm.commentsCount,
     isTextEditorEmpty: vm => !vm.comment.content?.trim(),
     discussion: vm => vm.$refs.discussion,
-    commentInput: vm => vm.$refs.commentInput,
+    commentEditor: vm => vm.$refs.commentEditor,
```

Comment:
If we are going to have it as computed lets have both or none. If not reused, we can just eliminate computed.